cobigen-cli/src/main/java/cobigen.bat

```diff
@@ -34,7 +34,7 @@ REM echo setting PATH
 REM setx PATH %JAVA_HOME%\bin;%PATH% -m
 REM echo Display java version
 REM java -version
-set CLASSPATH=.;C:\Program Files\Java\jdk1.8.0_181;C:\Program Files\Java\jdk1.8.0_181\lib\dt.jar;C:\MyData\IDE4\workspaces\com.cobigen.picocli\picocli-2.2.1.jar;C:\Users\syadav9\.m2\repository\com\devonfw\cobigen\openapiplugin\2.2.0\openapiplugin-2.2.0.jar
+set CLASSPATH=.;%JAVA_HOME%;C:\Program Files\Java\jdk1.8.0_181\lib\dt.jar;C:\MyData\IDE4\workspaces\com.cobigen.picocli\picocli-2.2.1.jar;C:\Users\syadav9\.m2\repository\com\devonfw\cobigen\openapiplugin\2.2.0\openapiplugin-2.2.0.jar
```

Comment:
Thanks for the change! However, I still see paths that depend on your computer, like `C:\MyData\IDE4\workspaces\com.cobigen.picocli\picocli-2.2.1.jar` or `C:\Users\syadav9\.m2\repository\com\devonfw\cobigen\openapiplugin\2.2.0\openapiplugin-2.2.0.jar`. There we should use relative paths.